.circleci/config.yml

```diff
@@ -30,18 +30,28 @@ jobs:
           keys:
             - composer<< parameters.cv >>-{{ checksum "composer.lock" }}
             - composer<< parameters.cv >>-
-      - run: COMPOSER_MEMORY_LIMIT=-1 composer update
-      - run: composer install -n --prefer-dist
+      - run:
+          name: "Update Composer"
+          command: COMPOSER_MEMORY_LIMIT=-1 composer update
+      - run:
+          name: "Composer install"
```

Comment:
previous `composer update` already did the install ....

Reply:
Thanks for pointing this out @jkowalleck - fix in PR #39